docker-compose example/docker-compose.example.yml

```diff
@@ -17,9 +17,21 @@ services:
       mariadb:
         condition: service_healthy
     ## Security options for some restricted systems
+    read_only: true
+    cap_drop:
+      - ALL
     security_opt:
-      - seccomp:unconfined
-      - apparmor:unconfined
+      - no-new-privileges:true
+      # not necessary since Docker Engine v20
+      #   - seccomp:unconfined
+      #   - apparmor:unconfined
+    ## Uncomment to enforce resource usage limits
+    # deploy:
```

Comment:
According to the https://docs.docker.com/compose/compose-file/deploy/, the `deploy` section is used only by Docker Swarm (something like Kubernetes, but from Docker).
We currently don't officially support Docker Swarm and for regular `docker compose` command it is ignored, so currently I don't see a value from adding this template here. Even more: it will create a false impression that there is a way to set some limits, while it isn't true.

Reply:
@kkovaletp I think the official documentation is misleading, the deploy section does impact docker compose's behaviour (without using Docker Swarm), if I set a low memory limit I see it getting it enforced by the OOM killer in case the container exceeds the threshold

Reply:
you can also confirm this by having a look at the `docker stats` output, the `LIMIT` column will correspond to the values set here

Reply:
Thanks, I'll test this as well